README.md

```diff
@@ -21,6 +21,52 @@ This repository contains the My Heart Counts iOS application, which is implement
+        - Note: you'll also need to edit the watch app's Info.plist and adjust the `WKCompanionAppBundleIdentifier` entry
+    - Select your own development team in the MyHeartCounts and MyHeartCountsWatchApp targets, and enable the automatic code signing option
+3. Adjust the app's run configuration (open via `cmd+shift+,`) and enable the following options:
+    - `--useFirebaseEmulator`
+        - If you wish to use a custom firebase deployment instead of a local emulator, you'll need to use the `--overrideFirebaseConfig plist=name` flag instead, where `name` is either an absolute path of a GoogleService-Info.plist file (this will only work in the simulator), or the filename (without extension) of a GoogleService-Info.plist file bundled with the app.
+    - `--overrideStudyBundleLocation`
+        - Specify the absolute path of the `.aar` file generated above
+        - Note: if you're running the app on a physical device, specifying the file location on the Mac won't work, since the iPhone can't access that. Instead, you can do one of the following:
+            - Bundle the study definition into the app:
+                - Drag the `.aar` file into the app's Resources folder in Xcode
+                - Adjust the code in the `StudyBundleLoader` type to simply always load from that in-bundle URL
+            - Host the study definition using the Firebase storage emulator:
+                - Open the Storage emulator (likely at http://localhost:4000/storage)
+                - Upload the `mhcStudyBundle.spezistudybundle.aar` file to the `/public` folder
+                - Configure the `--overrideStudyBundleLocation` argument to point to `http://HOSTNAME.local:9199/v0/b/myheart-counts-development.appspot.com/o/public%2FmhcStudyBundle.spezistudybundle.aar?alt=media`
+                    - Note that you'll need to replace `HOSTNAME` with your Mac's local-network name (you can find this in Settings.app → General → Sharing → Local hostname)
+    - `--disableAutomaticBulkHealthExport`
+        - (this option will disable the historical health data collection, improving performance when running the app on a real device)
+
+> [!NOTE]  
+> Please make sure not to commit and push any of the SensorKit, Code Signing, and run argument changes listed above; these changes are only required for local development.
+
+### The Firebase Environment
+1. Go to the firebase submodule: `cd MyHeartCounts-Firebase`
+2. Run `npm run prepare`
+3. Run `npm run serve:seeded`
+
+### The Study Definition
+1. Go to the definitions submodule: `cd MyHeartCounts-StudyDefinitions`
+2. Run `swift run MHCStudyDefinitionExporterCLI export ..` to generate a study bundle
+    - This will place a `mhcStudyDefinition.spezistudybundle.aar` file in the root of the MyHeartCounts-iOS repo
+    - you can have it saved elsewhere by replacing the `..` with the path of the folder where you want the study definition to be placed
```

Comment:
If we refer to the elements above we should probably reverse the order again and move these up.